lib/model/internal_link.dart

```diff
@@ -98,7 +98,7 @@ Uri narrowLink(PerAccountStore store, Narrow narrow, {int? nearMessageId}) {
     fragment.write('/near/$nearMessageId');
   }
 
-  return store.realmUrl.replace(fragment: fragment.toString());
```

Comment:
96593ba `internal_links [nit]: Add path: '/' to store.realmUrl.replace for proper Url handling`
I think we don't ever use `[nit]` in commit summaries, so we can just drop that.

Reply:
We don't have tests in this commit, and the tests added in the other commit do not fail if we remove this change. We need to do some testing here.